VCFv4.3.tex

```diff
@@ -399,19 +399,19 @@ \subsubsection{Genotype fields}
 
 \begin{itemize}
 \renewcommand{\labelitemii}{$\circ$}
-  \item AD, ADF, ADR: Per-sample read depths for each allele; total (AD), on the forward (ADF) and the reverse (ADR) strand. (Integer, Number=R)
-  \item DP : Read depth at this position for this sample. (Integer)
-  \item EC : Comma separated list of expected alternate allele counts for each alternate allele in the same order as listed in the ALT field. Typically used in association analyses. (Integers)
-  \item FT : Sample genotype filter indicating if this genotype was ``called'' (similar in concept to the FILTER field). Again, use PASS to indicate that all filters have been passed, a semi-colon separated list of codes for filters that fail, or `.' to indicate that filters have not been applied. These values should be described in the meta-information in the same way as FILTERs (String, no white-space or semi-colons permitted).
-  \item GQ : Conditional genotype quality, encoded as a phred quality $-10log_{10}$ p(genotype call is wrong, conditioned on the site's being variant). (Integer)
-  \item GP : Genotype posterior probabilities in the range 0 to 1 using the same ordering as the GL field; one use can be to store imputed genotype probabilities. (Float)
-  \item GT : Genotype, encoded as allele values separated by either of $/$ or $\mid$. The allele values are 0 for the reference allele (what is in the REF field), 1 for the first allele listed in ALT, 2 for the second allele list in ALT and so on. For diploid calls examples could be $0/1$, $1\mid0$, or $1/2$, etc. Haploid calls, e.g. on Y, male non-pseudoautosomal X, or mitochondrion, are indicated by having only one allele value. A triploid call might look like $0/0/1$. If a call cannot be made for a sample at a given locus, `.' must be specified for each missing allele in the GT field (for example `$./.$' for a diploid genotype and `.' for haploid genotype). The meanings of the separators are as follows (see the PS field below for more details on incorporating phasing information into the genotypes):
+  \item AD, ADF, ADR (Integer, Number=R): Per-sample read depths for each allele; total (AD), on the forward (ADF) and the reverse (ADR) strand.
```

Comment:
Why does this entry include the "Number" field? Let's be consistent, either they all do or they all don't.